PROTO_tests/tests/client_dtypes_test.py

```diff
@@ -0,0 +1,163 @@
+import ipaddress
+import random
+
+import pytest
+
+import arkouda as ak
+from arkouda import client_dtypes
+
+INT_TYPES = [ak.int64, ak.uint64]
+IPS = [3232235777, 2222222222, 1234567890, 7]
+
+
+class TestClientDTypeTests:
+    """
+    Note: BitVector operations are not tested here because the class is
+    only a wrapper on a pdarray to display as such.
+    The class does not actually store values as bit-values,
+    it only converts to a bit representation for display.
+    Thus, pdarray testing covers these operations.
+    """
+
+    @pytest.mark.parametrize("int_types", INT_TYPES)
+    def test_BitVector_creation(self, int_types):
```

Comment:
we should be consistent in our test names
```suggestion
    def test_bit_vector_creation(self, int_types):
```

Reply:
same goes for `test_Field_creation` below

Reply:
Updated